### PR TITLE
[css-text-3] Bikeshed improvements related to property values

### PR DESCRIPTION
**Pull request by SebastianZ**
This is my first PR, so please review carefully before merging!

Sebastian

### CODE REVIEW

File: css-text/Overview.bs [hunk 1]
```diff
@@ -2226,50 +1856,17 @@ Cursive Scripts</h4>
...
-        <tr>
-          <th>Animatable:
-          <td>as <a href="https://www.w3.org/TR/css3-transitions/#animtype-lpcalc">length, percentage, or calc</a>, but only if keywords match
-        <tr>
-          <th>Canonical order:
-          <td><abbr title="follows order of property value definition">per grammar</abbr>
-    </table>
+    <pre class="propdef">
+    Name: text-indent
+    Value: [ <<length>> | <<percentage>> ] && hanging? && each-line?
```
Comment (tabatkins) [inline css-text/Overview.bs]: Use `<<length-percentage>>` instead.

Comment (SebastianZ) [inline css-text/Overview.bs]: Done in https://github.com/w3c/csswg-drafts/pull/164/commits/9bc5074825ff8a68ce6e989af4196b0c28de3b70.

Sebastian

File: css-text/Overview.bs [hunk 2]
```diff
@@ -1898,38 +1582,16 @@ Possible Algorithms</h4>
...
-    <tr>
-      <th>Animatable:
-      <td>as <a href="https://www.w3.org/TR/css3-transitions/#animtype-lpcalc">length, percentage, or calc</a>
-    <tr>
-      <th>Canonical order:
-      <td>N/A
-  </table>
+  <pre class="propdef">
+  Name: word-spacing
+  Value: normal | <<length>> | <<percentage>>
```
Comment (tabatkins) [inline css-text/Overview.bs]: Wanna make this `<<length-percentage>>` too?

**Comment by SebastianZ** [comment]
I've changed both occurrances of `<<length>> | <<percentage>>` to `<<length-percentage>>` now in https://github.com/w3c/csswg-drafts/pull/164/commits/cea495a2550ee6c0db3872dfc0f46041e6abb02a.

Sebastian

**Comment by tabatkins** [comment]
Thanks!

**Pull request merged**